nvflare/app_opt/xgboost/histogram_based_v2/executor.py

```diff
@@ -149,6 +149,7 @@ def _notify_client_done(self, rc, fl_ctx: FLContext):
         """
         if rc != 0:
             self.log_error(fl_ctx, f"XGB Client stopped with RC {rc}")
+            self.system_panic(f"XGB Client stopped with non zero RC {rc}", fl_ctx)
```

Comment:
Do we need to call it here? If rc is not 0, is the system already aborted?

Reply:
When I am doing testing, I found there are a few situations where the RC return things other than 0, but the job continues.

It keep happening if we use "in_process=False" and there is trouble loading the plugin ".so" (due to either environment issue or typos)